CONTRIBUTING.md

```diff
@@ -59,6 +59,22 @@ During the collector release process, all `./.chloggen/*.yaml` files are transcr
 
 Alternately, copy `./.chloggen/TEMPLATE.yaml`, or just create your file from scratch.
 
+## Platform Agnostic Code
+
+In order to ensure compatibility with different operating systems, code should be platform-agnostic. Below are some guidelines to follow when writing platform-agnostic code:
```

Comment:
```suggestion
## Portable Code

In order to ensure compatibility with different operating systems, code should be portable. Below are some guidelines to follow when writing portable code:
```

I don't think the goal should be to be platform agnostic, but instead to be portable.  Some of the things we're doing are highly platform dependent.